tsconfig.base.json

```diff
@@ -107,6 +107,7 @@
       "@nx/storybook": ["packages/storybook"],
       "@nx/storybook/*": ["packages/storybook/*"],
       "@nx/typedoc-theme": ["typedoc-theme/src/index.ts"],
+      "@nx/ui-theme": ["graph/ui-theme/src/index.ts"],
```

Comment:
Should be `@nx/graph/ui-theme` to match the other libs and differentiate from `@nx/nx-dev`.